internal/restic/node_xattr_linux.go

```diff
@@ -0,0 +1,15 @@
+package restic
+
+import (
+	"syscall"
+
+	"github.com/pkg/xattr"
+)
+
+//isXattrNoData indicates if an error is NODATA
```

Comment:
The comment should explain why we're checking.

Reply:
updated